### PR TITLE
Update JWT install guide

### PR DESCRIPTION
**Pull request by jasonwilliams14**
### Proposed changes
This is an update to the JWT token method to install NGINX Ingress controller, providing a  more detailed walkthrough.

### Checklist
Before creating a PR, run through this checklist and mark each as complete.

- [x ] I have read the [CONTRIBUTING](https://github.com/nginxinc/kubernetes-ingress/blob/main/CONTRIBUTING.md) doc
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ x] I have checked that all unit tests pass after adding my changes
- [ x] I have updated necessary documentation
- [ x] I have rebased my branch onto main
- [ x] I will ensure my PR is targeting the main branch and pulling from my branch from my own fork

### CODE REVIEW

File: docs/content/installation/using-the-jwt-token-docker-secret.md [hunk 1]
```diff
@@ -18,51 +18,150 @@ This document explains how to use the NGINX Plus Ingress Controller image from t
 * For NGINX Ingress Controller, you must have the NGINX Ingress Controller subscription -- download the NGINX Plus Ingress Controller (per instance) JWT access token from [MyF5](https://my.f5.com).
 * To list the available image tags using the Docker registry API, you will also need to download the NGINX Plus Ingress Controller (per instance) certificate (`nginx-repo.crt`) and the key (`nginx-repo.key`) from [MyF5](https://my.f5.com).
 
+## Order of steps to pull NGINX Ingress controller from F5  registry
+
+1. Decide on what NGINX Ingress controller image you want to use. [NGINX Ingress controller images](https://docs.nginx.com/nginx-ingress-controller/technical-specifications/#images-with-nginx-plus "Available NGINX Ingress controller images")
+2. Log into MyF5 portal. [MyF5 portal login](https://myf5.com/ "MyF5 portal login")
```
Comment (dreesf5) [inline docs/content/installation/using-the-jwt-token-docker-secret.md:24]: Perhaps give more description here.  Something like:  You'll find the token under the "Subscription Detail" section of your subscription.

File: docs/content/installation/using-the-jwt-token-docker-secret.md [hunk 2]
```diff
@@ -18,51 +18,150 @@ This document explains how to use the NGINX Plus Ingress Controller image from t
...
+3. Download the JWT token from the `MyF5` portal.
+4. Create kubernetes secret using the JWT token that is provided in the MyF5 portal.
+   You can `cat` the contents of the JWT token and then store the output to use in the following steps. Make sure that there are no additional characters or extra whiespace that might have been accidently added. This will broke authorization and p   revent the NGINX Ingress controller image from being downloaded successfully.
```
Comment (dreesf5) [inline docs/content/installation/using-the-jwt-token-docker-secret.md:27]: s/broke/brake/.  Extra whitespace in prevent.

File: docs/content/installation/using-the-jwt-token-docker-secret.md [hunk 3]
```diff
@@ -18,51 +18,150 @@ This document explains how to use the NGINX Plus Ingress Controller image from t
...
+
+1. Clone the nginxinc/kubernetes-ingress repository
+2. Change directory into deployents/helm-chart of the recently cloned repo.
+3. Modify the `values.yaml` file.
+
+We want to set a few lines for NGINX Plus Ingress controller to be deployed.
+
+1. Change to `nginxplus`
+2. Specify NGINX Ingress controller image to use.
+3. Specify a `imagePullSecrets` to pull from the private registry.
```
Comment (dreesf5) [inline docs/content/installation/using-the-jwt-token-docker-secret.md:85]: imagePullSecretName

File: docs/content/installation/using-the-jwt-token-docker-secret.md [hunk 4]
```diff
@@ -18,51 +18,150 @@ This document explains how to use the NGINX Plus Ingress Controller image from t
...
+
+    ## The name of the service account of the Ingress Controller pods. Used for RBAC.
+    ## Autogenerated if not set or set to "".
+    # name: nginx-ingress
+
+    ## The name of the secret containing docker registry credentials.
+    ## Secret must exist in the same namespace as the helm release.
+    imagePullSecretName: regcred
+```
+
```
Comment (dreesf5) [inline docs/content/installation/using-the-jwt-token-docker-secret.md:120]: You should then write out the command line one would use for the manifests.

```bash
helm install my-release -n nginx-ingress -f values.yaml .
```

**Review by dreesf5 — CHANGES_REQUESTED** [review]
I tested both via sources and manifest and both worked now.  Small things to make it a bit more readable. 

I'd also be more clear about how and when to use the namespace -n nginx-ingress.

File: docs/content/installation/using-the-jwt-token-docker-secret.md [hunk 5]
```diff
@@ -18,51 +18,151 @@ This document explains how to use the NGINX Plus Ingress Controller image from t
...
+```yaml
+image:
+  ## The image repository of the Ingress Controller.
+  repository: private-registry.nginx.com/nginx-ic/nginx-plus-ingress
+
+  ## The tag of the
+  tag: 3.1.1
+```
+
+### Specify the `imagePullSecrets`
```
Comment (dreesf5) [inline docs/content/installation/using-the-jwt-token-docker-secret.md:106]: I would update this to `imagePullSecretName` as well.  Sorry I missed it the first time.

File: docs/content/installation/using-the-jwt-token-docker-secret.md [hunk 6]
```diff
@@ -18,51 +18,155 @@ This document explains how to use the NGINX Plus Ingress Controller image from t
...
+
+You can now install NGINX Ingress controller with the `values.yaml` file modified.
+
+```bash
+helm install nicdev01 -n nginx-ingress --create-namespace -f values.yaml .
+```
+
+The above command will install NGINX Ingress controller in the `nginx-ingress` namespace. If the namespace does not exist, `--create-namespace` will create it. Using `-f values.yaml` tells `helm` to use the `values.yaml` file that we modified earlier with the settings we want to apply for our NGINX Ingress controller deployment.
+
+### Using the `helm` charts method:
```
Comment (brianehlert) [inline docs/content/installation/using-the-jwt-token-docker-secret.md:130]: Is this an alternate Helm install method that is different than what begins on line 70?
Line 70 represents itself as THE Helm install method.  So it is kind of confusing to the novice to have this insert itself in as something that is different without much explanation.

Maybe an explanation like:  "Helm also supports overriding or defining values using the command line.  If you have an existing values files and want to override or test this method before committing to values files changes, you can insert the necessary values using the following single command line option."

File: docs/content/installation/using-the-jwt-token-docker-secret.md [hunk 7]
```diff
@@ -18,51 +18,155 @@ This document explains how to use the NGINX Plus Ingress Controller image from t
...
    {
     "name": "nginx-ic/nginx-plus-ingress",
     "tags": [
-        "3.1.1-alpine",
-        "3.1.1-ubi",
-        "3.1.1"
+        "3.1.0-alpine",
+        "3.1.0-ubi",
+        "3.1.0"
     ]
```
Comment (dreesf5) [inline docs/content/installation/using-the-jwt-token-docker-secret.md:152]: Did you really intend to change these to 3.1.0?

**Review by dreesf5 — CHANGES_REQUESTED** [review]

**Review by ADubhlaoich — APPROVED** [review]
I have made numerous rewrites in this PR to improve the readability of the page: it will need an additional once-over at the minimum to ensure that the instructions are accurate.

**Review by haywoodsh — APPROVED** [review]

**Review by dreesf5 — APPROVED** [review]

**Pull request merged**